libmenu/matemenu-tree.c

```diff
@@ -3265,8 +3264,8 @@ get_by_desktop (DesktopEntrySet *entry_pool,
     data.set      = set;
     data.category = desktop_name;
     desktop_entry_set_foreach (entry_pool,
-                              (DesktopEntrySetForeachFunc) get_by_desktop_foreach,
-                              &data);
+                                                   (DesktopEntrySetForeachFunc) get_by_desktop_foreach,
```

Comment:
Indent

Reply:
Is this still in the file? I have a lot of trouble following some of these inline comments(and they all send separate emails), and thought you all had resolved these indent issues. If so you might want to push a fix for the formatting and be done with it. I would be at risk of screwing that up at this point. Worst case we can revert

Reply:
No, this one is fixed.
When you add `.patch` or `.diff` to the commit link, than you see the raw-formated commit in browser.
https://github.com/mate-desktop/mate-menus/pull/86/commits/07bb366dc28e1039ae787b39120af24b422f39fb.patch
I will start with re-formatting all files of this repo to drop all shitty tabs from files, to avoid this annoying discussion "How to format with tabs in code" in nearly every PR.
So, no need to revert this PR for one line.
Believe me, i am so tired to review PRs when people have so much problems with that, and do not understand simple hints from reviewers.